front/lib/api/chat.ts

```diff
@@ -86,8 +86,10 @@ export async function getChatSession(
     title: chatSession.title,
     messages: messages.map((m) => {
       return {
+        id: m.id,
```

Comment:
There should not be any id here? The client should not know about the message id no?

Reply:
Can we remove it now?